docs/docs_skeleton/vercel.json

```diff
@@ -2952,6 +2952,46 @@
       "source": "/docs/modules/model_io/models/llms/integrations/writer",
       "destination": "/docs/integrations/llms/writer"
     },
+    {
+      "source": "/docs/modules/model_io/models/llms/integrations/amazon_api_gateway_example",
+      "destination": "/docs/integrations/llms/amazon_api_gateway"
+    },
+    {
+      "source": "/docs/modules/model_io/models/llms/integrations/azureml_endpoint_example",
```

Comment:
source should be the page it was just changed from, in this case `/docs/integrations/llms/azureml_endpoint_example`

Reply:
Got it.
If you name it as {"old": ..., "current or new": ...} it would be less confusing. "source" and "destination" are ambiguous.

Reply:
@baskaryan Are you sure about "source should be the page it was just changed from"?
I was using the L2952-2953 (and previous lines) as an example.
   {

> 
>       "source": "/docs/modules/model_io/models/llms/integrations/writer",
>       "destination": "/docs/integrations/llms/writer"
>     },

I can open the ["destination" page](https://python.langchain.com/docs/integrations/llms/writer) 
but the [source page](https://python.langchain.com/docs/modules/model_io/models/llms/integrations/writer) reroute me to the destination page.

Reply:
- [+]